### ISSUE TITLE
Configure logrotate for high log volume pods

### ISSUE THREAD

**Issue by surajssd** [kinvolk/lokomotive#1206]
Envoy shipped with contour generates humongous amounts of logs. We should have a strategy to rotate these logs.

Envoy can generate logs of hundreds of GBs. In cases where the pod is not daemonset it is evicted but in Envoy's case it is daemonset and can create downtime.

**Comment by surajssd** [comment]
Related to https://github.com/kinvolk/lokomotive/issues/1196

**Comment by invidian** [comment]
I thought Docker and/or Kubelet automatically rotate the logs already. Did you hit a case of very big log files, e.g. exhausting OS disk space?

**Comment by surajssd** [comment]
@invidian yep that's correct, the log file for envoy went upto 300GB. And kubelet cannot easily reconcile in such cases.
